### ISSUE TITLE
ERROR Group id, 999, already in use - exiting 

### ISSUE THREAD

**Issue by IZANAI** [boredazfcuk/docker-icloudpd#65]
hi, I've met the ERROR like this, how to solve this ERROR？THANKS!

`$ id wayne`
`uid=1002(wayne) gid=3001(everyone) groups=3000(Administrators),3001(everyone),999(docker)`

<img width="1322" alt="iShot2021-05-14 22 46 28 1" src="https://user-images.githubusercontent.com/19698554/118291261-f8e4a800-b509-11eb-962a-59a264d967e7.png">

**Comment by boredazfcuk** [comment]
Hi,

The group ID 999 is already in use by the group `ping`. You will need to either change the group ID of the user account, or set the environment variable `ForceGID` to `True`. This may have unintended consequences.

**Comment by pelargir** [comment]
I had to delete and re-create my container with the correct user, UID, group, and GID (all matching my Synology) before this would work. I tried simply changing the settings in the existing container but kept getting errors about the user or group already existing. Apparently, once the incorrect user or group has been created inside the container they will continue existing until the container is deleted and re-created. Not what I expected, but it did work.
